jwst/ramp_fitting/utils.py

```diff
@@ -855,11 +902,15 @@ def get_dataset_info(model):
 
     ngroups: int
         number of groups per integration
+
+    group_time: float
+        Time increment between groups, in seconds.
     """
 
     instrume = model.meta.instrument.name
     frame_time = model.meta.exposure.frame_time
     ngroups = model.meta.exposure.ngroups
+    group_time = model.meta.exposure.group_time
 
     n_int = model.data.shape[0]
     nreads = model.data.shape[1]
```

Comment:
`nreads` should be the same as `ngroups`.  If they're not the same, it would be better to log a warning and override the value of `ngroups` with `nreads`, which is more likely to be correct, since it's based on the image shape.

Reply:
Good point. Updated the code and comments to reflect that.